websites/#/대성마이맥/dist/metadata.json

```diff
@@ -11,10 +11,11 @@
 	"description": {
 		"ko_KR": "디지털대성은 최고 품질의 교육 콘텐츠를 제공하는 국내 대표 교육 기업입니다. 초ㆍ중ㆍ고 및 대입 수험생을 대상으로 온ㆍ오프라인 강의부터 테스트, 논술, 입시컨설팅 등 교육 토탈 서비스를 제공하고 있습니다.",
 		"en": "Digital Daesung is Korea's leading educational company that provides the highest quality educational content. It provides total education services to elementary, middle, high school and college students, from online and offline lectures to testing, essay writing, and college entrance consulting.",
-		"vi_VN": "Digital Daesung là công ty giáo dục hàng đầu Hàn Quốc, cung cấp nội dung giáo dục chất lượng cao. Chúng tôi cung cấp dịch vụ giáo dục toàn vẹn cho bậc tiểu học, trung học, cao đẳng và đại học, từ học trực tuyến đến trực tiếp, luyện viết luận và tư vấn nộp hồ sơ đại học."
+		"vi_VN": "Digital Daesung là công ty giáo dục hàng đầu Hàn Quốc, cung cấp nội dung giáo dục chất lượng cao. Chúng tôi cung cấp dịch vụ giáo dục toàn vẹn cho bậc tiểu học, trung học, cao đẳng và đại học, từ học trực tuyến đến trực tiếp, luyện viết luận và tư vấn nộp hồ sơ đại học.",
+		"nl": "Digital Daesung is het vooraanstaande Koreaanse onderwijsbedrijf dat onderwijsinhoud van de hoogste kwaliteit levert. Het bedrijf biedt totale onderwijsdiensten aan basisschool-, middelbare school- en universiteitstudenten, van online en offline colleges tot toetsen, essays schrijven en advies voor toelating tot de universiteit."
```

Comment:
`basisschool-, middelbare school- en universiteitstudenten,`
basisschool en middelbare ben je niet echt student maar scholier.

en `van online en offline colleges tot toetsen,`
colleges is volgens mij niet het beste heirvoor... @QkeleQ10